python-avd/pyavd/_eos_designs/schema/schema_fragments/platform_settings.schema.yml

```
@@ -131,6 +160,188 @@ keys:
               type: bool
               default: false
               description: Support for all-active EVPN gateway redundancy.
+            hardware_counters:
+              type: dict
+              description: |-
+                Support for enabling counters for features using programmable hardware counter resources.
+                Feature will be ignored on unsupported platforms.
+              keys:
+                supported:
+                  type: bool
+                  description: |-
+                    General supportability of the `hardware counter` feature.
+                    Setting this key to `false` for the specific platform will disable support of any specific hardware counter functionality (`feature`, `drop`, etc.) for this platform.
+                  default: true
+                feature:
+                  type: dict
+                  keys:
+                    supported:
+                      type: bool
+                      description: |-
+                        General supportability of the `hardware counter feature` feature.
```

I am not a native English speeaker but supportability feels a bit clunky not sure if my suggestion is better though :)
```suggestion
                        General support of the `hardware counter feature` feature.
```

---

```suggestion
                        General support for the `hardware counter` feature.
```

---

Revised wording but suggested `scope` may not accurately reflect real impact.
Please check if updated description makes sense